virtwho/__init__.py

```diff
@@ -9,6 +9,10 @@
 
 HYPERVISOR_FILE = f'/etc/virt-who.d/{HYPERVISOR}.conf'
 
+DEFAULT_HYPERVISOR_FILE = '/etc/virt-who.d/virtwho-config.conf'
+
+SECTION_NAME = 'virtwho-config'
```

Comment:
Looks the two variables are not so readable, could we change to 
```
SECOND_HYPERVISOR_FILE = f'/etc/virt-who.d/{HYPERVISOR}-second.conf'

SECOND_HYPERVISOR_SECTION = 'virtwho-{HYPERVISOR}-second'
```

Reply:
Changed